docs/maui/views/AvatarView.md

```diff
@@ -0,0 +1,360 @@
+---
+title: AvatarView - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "AvatarView is a control for displaying an image or user's initials avatar."
```

Comment:
Perhaps something like:

```suggestion
description: "AvatarView is a control for displaying a user's avatar image or their initials."
```

Reply:
@bijington A much better wording.  I'll also use this in the code base too.